src/Framework/Analyzer/src/MinimalActions/DiagnosticDescriptors.cs

```diff
@@ -0,0 +1,20 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using Microsoft.CodeAnalysis;
+
+namespace Microsoft.AspNetCore.Analyzers.MinimalActions
+{
+    [System.Diagnostics.CodeAnalysis.SuppressMessage("MicrosoftCodeAnalysisReleaseTracking", "RS2008:Enable analyzer release tracking")]
+    internal static class DiagnosticDescriptors
+    {
+        internal static readonly DiagnosticDescriptor DoNotUseModelBindingAttributesOnMinimalActionParameters = new(
+            "ASP0003",
+            "Do not use model binding attributes with Map actions",
+            "{0} should not be specified for a {1} delegate parameter",
+            "Usage",
+            DiagnosticSeverity.Warning,
+            isEnabledByDefault: true,
+            helpLinkUri: "https://aka.ms/minimal-action/analyzer");
```

Comment:
What is this link going to point to? And should we remove minimal from it

Reply:
```suggestion
            helpLinkUri: "https://aka.ms/aspnet-app/analyzers");
```

How about this? The plan is to have it point to some docs like the ones NetAnalyzers does.

Reply:
Sounds good, and we'll eventually have more analyzers not minimal specific that can go there too.